neqo-transport/src/recv_stream.rs

```diff
@@ -523,7 +523,7 @@ mod tests {
         let flow_mgr = Rc::new(RefCell::new(FlowMgr::default()));
         let conn_events = ConnectionEvents::default();
 
-        let mut s = RecvStream::new(567.into(), 1024, flow_mgr.clone(), conn_events.clone());
+        let mut s = RecvStream::new(567.into(), 1024, flow_mgr, conn_events);
```

Comment:
I'm not sure that I follow this change.  These are `Rc<>` instances that presumably we want to clone for passing into multiple invocations of this function.  Has `Rc` picked up an implementation of `Copy` in a recent version of rust?

I would have thought that moving to `Rc::clone(&flow_mgr)` would be wise, but I don't see that here either.

Reply:
Oh nice catch, I'll move it to Rc::clone() to make it more explicit.
I think I got fooled by the linter, which advises to remove .clone(), which doesn't make sense in an Rc context.

It might be worth opening an issue to clippy.